test/helpers/kubectl.go

```diff
@@ -143,6 +143,8 @@ var (
 		"global.gke.enabled":          "true",
 		"global.nativeRoutingCIDR":    "10.0.0.0/8",
 		"global.hostFirewall":         "false",
+		"config.ipam":                 "kubernetes",
+		"global.devices":              "", // Override "eth0 eth0\neth0"
```

Comment:
But where does the `eth0 eth0\neth0` come from? Presumably somewhere else in the CI code, right?

Reply:
#11969

Reply:
My guess is that the effect of setting 'global.devices' is not tested on the GKE datapath configuration.

Reply:
Looks like it yeah. I'm OK with this workaround for now, but /cc @pchaigno for figuring out how we want to tackle this in v1.9.

Reply:
The incorrect `global.devices` will be fixed by #12176. We should probably revert the devices overwrite once it's merged.